src/util.js

```diff
@@ -244,3 +244,20 @@ export function unquote_cmdline(text) {
 
     return words;
 }
+
+/*
+ * Return true if first argument is newer or equal version than the second argument.
+ * Both arguments are required to be strings, semver, wher delimiter is '.'.
+ */
+export function isVersionNewerOrEqual(v1, v2) {
```

Comment:
FTR, a standard comparison function returns -1, 0, or 1. Like cockpit's `compare_versions()` in pkg/storaged/utils.js.

It's ok like that, but just in case you want to keep the code in sync.

Reply:
nice, had no idea we have this implemented. I'll  take the code from there.